src/saleor_app/app.py

```diff
@@ -53,6 +63,20 @@ def include_webhook_router(self):
                 401: {"description": "Incorrect signature"},
                 404: {"description": "Incorrect saleor event"},
             },
+            dependencies=[
+                Depends(verify_saleor_domain),
+                Depends(verify_webhook_signature),
+            ],
+            route_class=WebhookRoute,
         )
+
         router.post("", name="handle-webhook")(handle_webhook)
+        self.webhook_handler_routes = {
+            name: APIRoute(
+                "",
+                endpoint,
+            )
+            for name, endpoint in self.webhook_handlers
+            if endpoint
+        }
         self.include_router(router)
```

Comment:
TODO: add the possibility to overload this whole thing, example use case would be to handle every webhook with one APIRoute handler and within that deal with the distribution of tasks (i.e. with a event journal system)